compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/RenderingUIView.uikit.kt

```diff
@@ -111,14 +110,15 @@ internal class RenderingUIView(
             onAttachedToWindow?.invoke()
             _isReadyToShowContent.value = true
         }
+        updateMetalLayerSize()
```

Comment:
This function is also called when moved _out_ of window (so the window becomes null, need to check that here before calling `metalLayerResize again`?). `UIWindow.screen` is nonnull, so starting function with a `let window = window ?: return` should do the trick?

Reply:
Done